packages/view-types/spatial/src/SpatialSubscriber.js

```diff
@@ -400,6 +401,13 @@ export function SpatialSubscriber(props) {
     obsCentroids, obsCentroidsIndex,
   ]);
 
+  const resolutionFilteredImageLayerLoaders = useMemo(() => {
+    // eslint-disable-next-line max-len
+    const shouldUseFullData = (ll, index) => useFullResolutionImage[meta[index].name] && Array.isArray(ll.data);
+    // eslint-disable-next-line max-len
```

Comment:
```suggestion
    const shouldUseFullData = (ll, index) => Array.isArray(useFullResolutionImage) && useFullResolutionImage.includes(meta[index].name) && Array.isArray(ll.data);
    // eslint-disable-next-line max-len
```

Can `useFullResolutionImage` be a list of the image names?

Also - would you be able to add a comment about the reasoning behind what the `useMemo` is doing, for future reference / other developers?